addons/account/views/account_tax_views.xml

```diff
@@ -271,6 +271,7 @@
                                 <field name="country_id"/>
                                 <field name="company_id" groups="base.group_multi_company"/>
                                 <field name="sequence"/>
+                                <field name="label" string="Label on PoS Receipts" groups="base.group_no_one"/>
```

Comment:
Setting the tax group label manually is considered as an advanced setting/feature. A user shouldn't have to touch it, since l10n module should pre fill the field automatically in most cases.